lib/crystalball/map_generator/parser_strategy.rb

```diff
@@ -21,6 +23,7 @@ def initialize(root = Dir.pwd, pattern:)
       def after_register
         files_to_inspect.each do |path|
           processor.consts_defined_in(path).each do |const|
+            yield const, path if block_given?
```

Comment:
do you still need it?

Reply:
Not right now, I guess I'll just remove it.